packages/plugin-commands-init/src/init.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs'
+import path from 'path'
+import { docsUrl } from '@pnpm/cli-utils'
+import { UniversalOptions } from '@pnpm/config'
+import PnpmError from '@pnpm/error'
+import writeProjectManifest from '@pnpm/write-project-manifest'
+import renderHelp from 'render-help'
+
+export const rcOptionsTypes = cliOptionsTypes
+
+export function cliOptionsTypes () {
+  return {}
+}
+
+export const commandNames = ['init', 'create', 'innit']
```

Comment:
```suggestion
export const commandNames = ['init', 'create']
```

We don't need this, although sometimes we need to make pnpm's behaviour consistent with npm, but this shouldn't contain "dross".

<img width="844" alt="image" src="https://user-images.githubusercontent.com/8198408/157225874-0cd2ca41-45b5-44b3-9f70-be26bae299a1.png">

There is currently only one public project in github that uses this.